packages/manager/modules/pci/src/projects/project/sidebar/sidebar.controller.js

```diff
@@ -83,6 +96,20 @@ export default class SidebarController {
     });
   }
 
+  getMe() {
+    return this.OvhApiMe.v6().get().$promise;
```

Comment:
My opinion is the more independent a component is, the better. Since the component is able to get the user using the APIs, I thought that is better way to do. Please let me know if you think otherwise :)

Reply:
So in this case we will call `/me` multiple times to retrieve the same response from the API, IMO it's better to inject resource

Reply:
This is done. Thanks for all your feedback :)